packages/react/src/auto/hooks/useHiddenInput.tsx

```diff
@@ -0,0 +1,25 @@
+import { useEffect } from "react";
+import type { Control } from "react-hook-form";
+import { useController, useFormContext } from "react-hook-form";
+import { useFieldMetadata } from "./useFieldMetadata.js";
+
+export const useHiddenInput = (props: { field: string; value: any; control?: Control<any> }) => {
+  const { field, value, control } = props;
+  const { path, metadata } = useFieldMetadata(field);
+  const form = useFormContext();
+
+  const { field: fieldProperties } = useController({
+    name: path,
+    rules: { required: metadata.requiredArgumentForInput },
+    control,
+  });
```

Comment:
Is this hook needed if it's values are never used? Tests still pass if it's removed.

It would provide `name,onBlur,onChange,ref,value,disabled`, but I don't think any of those would be relevant for a hidden input